packages/wonder-blocks-tooltip/src/components/__docs__/tooltip.stories.js

```diff
@@ -50,16 +54,34 @@ Default.args = {
     children: "some text",
 };
 
+Default.play = async ({canvasElement}) => {
+    // Arrange
+    // NOTE: Using `body` here to work with React Portals.
+    const canvas = within(canvasElement.ownerDocument.body);
+
+    // Act
+    // Triggers the hover state
+    await waitFor(async () => {
+        await userEvent.hover(canvas.getByText("some text"));
+    });
```

Comment:
**question:** Does it take a while for `"some text"` to show up?  Would the following work?
```
const text = await canvas.findByText("some text");
await userEvent.hover(text);
```

Reply:
hmmm good point! that should work. Let me change it.

Reply:
It worked, thanks for the suggestion 🎉